### ISSUE TITLE
Main slider captions overlapping on page load

### ISSUE THREAD

**Issue by eri-trabiccolo** [Nikeo/customizr#1293]
Reported here:
https://secure.helpscout.net/conversation/462551520/34125?folderId=607709

I cannot reproduce it on my end, though is visible on the user's website.
What is actually weird is that the slides should not be visible because of the loader overlapping them.
But it seems that in this case the slider loader appears with a certain delay. Most likely due to the caching/concatenating plugin in place which loads some heavy js a the very top of the page.
**Note:** The slider loader is hidden when the html tag has the class 'no-js' (default), it appears only when this class is replaced with 'js'. We perform the html class replacement here:
https://github.com/presscustomizr/customizr/blob/v4.0.11/core/init-base.php#L904

This is something that can actually happen for the way the main slider's slides are made.
Basically the `.carousel-caption`s are absolutely positioned, and they refer to the first positioned (not static) ancestor, which, when flickity is not enabled, is the `.czr-slider-holder`, not the `.carousel-cell`s.
**Note:** `.carousel-cell`s, on load, having a static position and width 100%, are displayed one of top of the other, but just the first one is visible, as the slider wrapper hides what overflows.

## Solution?
So the solution would be to set the `.czr-carousel .carousel-cell` position relative (flickity then, once enabled, will set the position to absolute).

This way, the cells will still stay one on top of the other, but each caption position will refer to the cell, hence only the first caption will be visible, as it should.
